packages/terra-demographics-banner/lib/DemographicsBanner.scss

```diff
@@ -0,0 +1,89 @@
+@import '~terra-mixins';
+@import './variables';
+
+.terra-DemographicsBanner {
+  background: $demographics-background;
+  color: $demographics-foreground;
+  display: flex;
+  flex-direction: row;
+  margin-bottom: 0.714em;
+  padding: 0.714em;
+
+  @media screen and (max-width: 767px) {
```

Comment:
Thanks brett! That did the trick! https://github.com/cerner/terra-core/pull/280/commits/841a6cd5ade3eeeaee81fd4b2bad787e972c0cc2